src/EditorFeatures/CSharpTest/Squiggles/ErrorSquiggleProducerTests.cs

```diff
@@ -122,27 +124,54 @@ void Test()
                     }
                 };
 
+            var diagnosticsAndSpans = await _producer.GetDiagnosticsAndErrorSpans(workspace, analyzerMap);
+
             var spans =
-                (await _producer.GetDiagnosticsAndErrorSpans(workspace, analyzerMap)).Item2
-                    .OrderBy(s => s.Span.Span.Start).ToImmutableArray();
+                diagnosticsAndSpans.Item1
+                    .Zip(diagnosticsAndSpans.Item2, (diagnostic, span) => (diagnostic, span))
+                    .OrderBy(s => s.span.Span.Span.Start).ToImmutableArray();
```

Comment:
![image](https://user-images.githubusercontent.com/201340/80550341-ab37e480-8974-11ea-8eee-5298d90d85df.png)

Reply:
I'll see if I can add a fourth one for balance.

Reply:
@sharwell Good, I was bugged by it not being in proper rhythm.